demos/using-dev-build/README.md

```diff
@@ -1,9 +1,17 @@
+
+[comment]: # This document is mainly written to be viewed on https://internetcomputer.org/docs/current/developer-docs/integrations/internet-identity/integrate-identity/
```

Comment:
did you mean to make an admonition?

> **Note**
> foo bar

Reply:
No, this is supposedly how to do comments in markdown: https://stackoverflow.com/questions/4823468/comments-in-markdown